core/citrus-spring/src/main/java/org/citrusframework/config/xml/StartServerActionParser.java

```diff
@@ -75,11 +76,13 @@ public void setServer(Server server) {
         }
 
         @Override
+        @Nullable
```

Comment:
a builder can return `null`? or is this because of overrides?

Reply:
No and yes. This is because of some annoying IDE warning and because it is Nullable from the base interface.
I remember that i added it at some places which took care of the IDE warning. At least in this particular case the warning is still present. I removed the Nullable because here for sure no null can be returned.